olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -341,6 +342,12 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    */
   bool RemoveFromCache(const geo::TileKey& tile);
 
+  client::CancellationToken GetListVersions(VersionsRequest request,
+                                            VersionsResponseCallback callback);
+
+  client::CancellableFuture<VersionsResponse> GetListVersions(
+      VersionsRequest request);
+
```

Comment:
This is a catalog specific API and does not belong to a layer. Also, it should be called `ListVersions` as **list** is already a verb and does not need help from **get** to describe the action. The meaning is _list the available catalog versions_ and not _get the list of catalog versions_.